src/clusters.ts

```diff
@@ -72,7 +72,7 @@ export async function getClusterDetails(vendorPortalApi: VendorPortalApi, cluste
       }
     
       const body: any = JSON.parse(await res.readBody());
-      if (clusters.length == body.totalClusters || body.clusters.length == 0) {
+      if (clusters.length == body.totalClusters || body.clusters == undefined) {
```

Comment:
is it possible to also leave the empty array check in? `clusters.length == body.totalClusters || body.clusters == undefined || body.clusters.length == 0`? this way if it's returned, but empty, it still works.